docs/products/opensearch/howto/opensearch-search-and-python.rst

```diff
@@ -0,0 +1,471 @@
+                            "operator": "and"
+                          }
+                        }
+                      }
+                    }
+      resp = client.search(index=INDEX_NAME, body=query_body)
+      log_titles(resp)
+
+You can run yourself the code to explore the ``match`` function. For example, if you want to find spring recipes in the title:
+
+.. code-block:: shell
+
+  python search.py match "title" "Spring"
+  [
+    'Spring Fever ',
+    'Spring Rolls ',
+    'Spring Feeling ',
+    'Spring Fever ',
+    'Spring Rolls ',
+    'Spring Feeling ',
+    'Spring Vegetable Sauté ',
+    'Spring-Onion Cocktail ',
+    'Braised Spring Legumes ',
+    'Asian Spring Rolls '
+  ]
+
+Find out more about `Match query <https://opensearch.org/docs/latest/opensearch/query-dsl/full-text/#match>`_.
+
+Multi match query
+------------------
+One useful query when you want to align the ``match`` query properties but expand it to search in more fields is the ``multi_match`` query. You can add several fields in the ``fields`` property, so we will be searching the ``query`` string across all those fields included in the list.
+
+.. code-block:: python
+
+     query_body = {
+                    "query": {
+                      "multi_match": {
+                        "query": query,
+                        "fields": [field1, field2 ...]
+                      }
+                    }
+                  }
+
+In the next query we are looking across the ``title`` and ``ingredients`` fields for recipes with "Summer" on them. 
+
+.. code-block:: python
+
+     query_body = {
+                    "query": {
+                      "multi_match": {
+                        "query": "Summer",
+                        "fields": [
+                          "title",
+                          "ingredients"
+                        ]
+                      }
+                    }
+                  }
+    resp = client.search(index=INDEX_NAME, body=query_body)
+    log_titles(resp)
+
+Check out more results for the ``multi_match`` queries creating your own ``multi_match`` query, for example
+
+::
+
+  python search.py multi_match title ingredients lemon
+
+
+Check out more about `multi match query <https://opensearch.org/docs/latest/opensearch/query-dsl/full-text/#multi-match>`_.
+
+Match phrase query
+------------------
+This query can be used to match phrases in a field. Where the ``query`` is the phrase that is being searched in a certain field:
+
+.. code-block:: python
+
+     query_body = {
+                    "query": {
+                      "match_phrase": {
+                        field: {
+                          "query": query
+                        }
+                      }
+                    }
+                  }
+
+If we are looking for a certain phrase, for example, ``pannacotta with lemon marmalade`` in the title, we may use a query like:
+
+.. code-block:: python
+
+     query_body = {
```

Comment:
I think I'd omit repeating almost the same query and move on sharing the example command line.